src/components/cylc/workspace/Lumino.vue

```diff
@@ -114,21 +113,35 @@ const resizeObserver = new ResizeObserver(() => {
   boxPanel.update()
 })
 
-onMounted(() => {
+const layoutsCache = window.caches.open('workspace-layouts')
```

Comment:
I think caches are tied to the domain. Because of Jupyter Server and Jupyter Hub services, there can be multiple web apps served from the same domain.

We should probably namespace our caches to avoid collisions, e.g:

```suggestion
const layoutsCache = window.caches.open('cylc/workspace-layouts')
```

The MDN docs for caches use a version identifier (i.e. `v1`) which might be a good idea (e.g. `cylc/workspace-layouts/v1`), but we could always add the version number later (i.e. for `v2`) if we need it.

Reply:
Yes (as suggested) and yes (understood).

Reply:
@MetRonnie, I still think it's a good idea to namespace our caches right?